cypresse2e/cypress/support/commands.js

```diff
@@ -24,9 +24,16 @@ export const randomString = (length) => {
 };
 
 Cypress.Commands.add('generateRandomEmail', () => {
-    return randomString(8) + '.' + randomString(8) + '@' + randomString(5) + '.com';
+    return randomString(64) + '@' + randomString(63) + '.com';
```

Comment:
@JayashreeSamantasinghar 
Maybe here we can pass parameters. It will give us more flexibility. Not in all tests we need 64 and 63 characters. We can pass parameters. Something like this:

`Cypress.Commands.add('generateRandomEmail', (num1, num2) => {`

Reply:
@galeza 
I did the changes here and changed accordingly in joinNow.spec.js and signIn.spec.js as well where it was used
Please review this

Reply:
@jessica-lau 
Here I think the randomString length for email body should not exceed  than 64 characters and domain should not exceed than 63 characters. As we took  
"export const randomString = (length) => {   "also  "for (var i = 0; i < length; i++) {"
so here we have to specify the length to a specific number else it will generate infinite numbers

Reply:
@jessica-lau 
Now I got it and took  character length in joinNow.spec